READMEs/consume-flow.md

```diff
@@ -41,7 +41,11 @@ Then, please refer to [publish-flow](publish-flow.md) and complete the following
 Now, you're Bob the data consumer.
 In usual cases, Bob buys the dataset but here, let's have Alice send hhim some tokens directly.
 
-In the same Python console as before:
+In the same Python console as before, first define the FACTORY_DEPLOYER_PRIVATE_KEY, needed to mint fake OCEAN for testing with ganache
+
+`export FACTORY_DEPLOYER_PRIVATE_KEY=0xc594c6e5def4bab63ac29eed19a134c130388f74f019bc74b8f4389df2837a58`
```

Comment:
I shortened the explanation but I can't integrate the export inside the Python code block, because our generated readmes pick up code blocks and can not contain both bash and python code. However, I think I've added a good compromise solution :D